api/envoy/config/core/v3/grpc_service.proto

```diff
@@ -58,12 +58,10 @@ message GrpcService {
     google.protobuf.UInt32Value max_receive_message_length = 4;
 
     // This provides gRPC client level control over envoy generated headers.
-    // For ``x-envoy-internal`` and ``x-forwarded-for`` header:
-    // If true, the header will be sent. But it can be overridden by setting the ``send_internal`` or ``send_xff``
-    // to false in ``Http::AsyncClient::StreamOptions``, as per stream-level control.
-    // If false, the header will be removed and can not be overridden by per stream option.
-    // Default to true.
-    google.protobuf.BoolValue send_envoy_generated_header = 5;
+    // If false, the header will be sent but it can be overridden by per stream option.
+    // If true, the header will be removed and can not be overridden by per stream option.
+    // Default to false.
```

Comment:
Looks good now, but it can just be regular `bool` as it defaults false now.

Reply:
Yeah I have thought about this, but was thinking we probably want the flexibility to override it to true in the future? 

Changing to `bool` certainly works for me as well. Updated.